### ISSUE TITLE
macOS flags BarTranslate as malware

### ISSUE THREAD

**Issue by trinity420** [ThijmenDam/BarTranslate#35]
After a restart suddenly macOS states "BarTranslate.app harms your computer. It is recommended to move it to the trash bin." translated from German. With a checkbox to "report malware to Apple".

**Comment by ThijmenDam** [comment]
I am aware of the issue and looking into it. Keep an eye on the releases page.

**Comment by AndroidBaaaby** [comment]
Hi @ThijmenDam, 1.1.0 didn't resolve this issue in Sonoma 14.2.

**Comment by ThijmenDam** [comment]
Alright, that sucks; I'd probably have to sign the app using a paid Apple developer account, which I don't have. Let me look into it a bit more, and if that's the only option... then I'll pay.

**Comment by michaelwills** [comment]
At this point even Chrome won't download it! That's surprising.

**Comment by ThijmenDam** [comment]
At this moment I unfortunately do not have the resources to sign the app using a paid developer account. 

**This means that BarTranslate, at this time, is NOT usable** because macOS flags it as malware.

**Comment by odysseus90210** [comment]
It appears to run, but isn't functional under Mac OS Sonoma 14.4 :(

**Comment by ThijmenDam** [comment]
> It appears to run, but isn't functional under Mac OS Sonoma 14.4 :( 

What do you mean with "non functional"?

**Comment by odysseus90210** [comment]
It is working after all. Sorry for the false alarm!

—Marc

**Comment by viktorhanacek** [comment]
You need to try to open the app, then go to Settings » Privacy & Security » scroll down and click Open anyway. And believe that @ThijmenDam is not doing somethin' 👹👹👹

**Comment by ThijmenDam** [comment]
> You need to try to open the app, then go to Settings » Privacy & Security » scroll down and click Open anyway. And believe that @ThijmenDam is not doing somethin' 👹👹👹

I develop this open source application in my free time. Feel free to open a pull request if my timeline does not fit your needs. 

That being said, from this April I will start working one day less each week. This day will be spent working on my personal projects (including BarTranslate), so you can expect development to speed up soon... 

Moreover, please consider that resolving this issue requires an active Apple Developer account that requires me to [pay a fee](https://developer.apple.com/support/compare-memberships/) every year. Consider donating if resolving this issue is a priority to you.

**Comment by viktorhanacek** [comment]
Of course, this was not meant in any bad way. Probably just a stupid joke :) I just wanted to show that the Apple protection has a reason and can be bypassed this way (and your app can be used thanks to this workaround).

But it can be bypassed this way for any application so people need to be aware that other people can have bad intentions.

**Comment by jamtur01** [comment]
@viktorhanacek - as a fellow open source maintainer, it's not a very funny joke.

**Comment by michaelwills** [comment]
I just tried on Sonoma 14.4. Initially it didn't work but after a reboot it runs. 👍 

I use an Alfred App workflow to remove the quarantine.

**Comment by ThijmenDam** [comment]
Issue has since been resolved. 

More robust alternative is downloading the App Store variant: https://apps.thijmendam.nl/bartranslate/
